results/config/api.yaml

```diff
@@ -28,9 +28,50 @@ spec:
       containers:
       - name: tekton-results-api
         image: ko://github.com/tektoncd/experimental/results/cmd/api
+        # Uncomment to use CloudSQL proxy defined below.
+        # args:
+        # - "-db_addr=127.0.0.1:3306"
+        env:
+        - name: DB_USER
+          valueFrom:
+            secretKeyRef:
+              name: mysql-db-creds
+              key: mysql-db-user
+        - name: DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: mysql-db-creds
+              key: mysql-db-password
 
+      # Uncomment to configure CloudSQL proxy.
```

Comment:
Is this something we intend on using in the future, or should we just delete this?

Reply:
I think it's nice to demonstrate that other non-local MySQL databases can be used. I might just move it wholesale into a separate install doc, in a future change.